src/model/scripts/lighting_training/train.py

```diff
@@ -30,6 +30,14 @@
         default="auto",
         help="dp | ddp | ddp_spawn | xla | deepspeed | fsdp",
     )
+    parser.add_argument("--stage", type=int, default=2, help="stage of deepspeed")
+    parser.add_argument(
+        "--offload_opt", action="store_true", help="offload optimizer of deepspeed"
+    )
+    parser.add_argument(
+        "--offload_par", action="store_true", help="offload parameters of deepspeed"
```

Comment:
Should throw error if not deepspeed but offload parameter/opt is true

Reply:
done I add it in utils.py to raise error when strategy isn't deepspeed.